shared/login/welcome/index.render.native.js

```diff
@@ -1,11 +1,12 @@
+// NOT FLOW - redo this
```

Comment:
these screens are crazy old and don't conform to dz2 at all, we'll have to redo them